### DISCUSSION TITLE
How do you handle boolean and numeric .env variables using dotenv?

### DISCUSSION THREAD

**Discussion by ajcastro** [quasarframework/quasar#9619] [General - Components / Directives / etc]
I saw from the docs that the latest recommended dotenv plugin as show in example is https://github.com/motdotla/dotenv, but when I tried it, it doesn't support boolean and numeric values. 

I reported an issue in their repo, https://github.com/motdotla/dotenv/issues/544. 

How do you handle boolean and numeric .env variables?

**Comment by metalsadman** [comment]
env variables always returns strings, you can do some checking ie. `process.env.IS_BOOL && process.env.IS_BOOL === 'true'` etc.. or use some parser library such as https://www.npmjs.com/package/dotenv-parse-variables.

**Reply by ajcastro** [reply]
I see. So that's how they do it in javascript land. I came from Laravel/PHP where .env are automatically parsed by default.
Thanks for the tip @metalsadman !
